### ISSUE TITLE
Table item width putting single characters on new line

### ISSUE THREAD

**Issue by sahaka** [qooob/authentic-theme#207]
There are many places where the last letter/character of text in a table icon is put on a new line. 

<img width="321" alt="screenshot 2015-07-24 15 58 52" src="https://cloud.githubusercontent.com/assets/4749006/8883466/493dfdf4-321d-11e5-8e1b-9bec6ff4ab3e.png">

<img width="1434" alt="screenshot 2015-07-24 16 02 14" src="https://cloud.githubusercontent.com/assets/4749006/8883546/7b510c6e-321d-11e5-9f5f-d528be4b8409.png">

Is there a way to fix this that I'm not aware of because overtime this small issues becomes very frustrating?

**Comment by qooob** [comment]
Yes, good point. So far, open theme settings and by going to theme extensions, add the following to CSS extension:

```CSS
.row.icons-row .icons-container a {
    word-break: normal;
    overflow: hidden;
}
```

On the next minor bug fixes release, I will add it to the distro.

**Comment by qooob** [comment]
It will be fixed in version 14.01.
